### PR TITLE
Few patch to correct git files

### PR DESCRIPTION
**Pull request by jeromecoutant**
Hi

- .mbed files are not useful to be in git
- .gitignore is updated

### CODE REVIEW

**Review by paul-szczepanek-arm — APPROVED** [review]

**Comment by paul-szczepanek-arm** [comment]
All PRs need to target development. But I'm rebasing the tracing PR and I'll include this as well.

File: .gitignore [hunk 1]
```diff
@@ -35,9 +35,10 @@ GettingStarted.html
...
 # mbed build system
 mbed-os/
+mbed-os-ble-utils/
 mbed_settings.py
 mbed_config.h
 *.pyc
 BUILD/
 cmake_build/
-
+.mbedls*
```
Comment (paul-szczepanek-arm) [inline .gitignore:44]: what is this file?

Comment (jeromecoutant) [reply .gitignore:44]: mbedls mock files :-)
https://github.com/ARMmbed/mbed-os-tools/blob/master/packages/mbed-ls/README.md#mocking-renaming-platforms

**Comment by paul-szczepanek-arm** [comment]
Rolled into https://github.com/ARMmbed/mbed-os-example-ble/pull/352

**Pull request closed without merging**